dotCMS/src/integration-test/java/com/dotcms/concurrent/DotConcurrentFactoryTest.java

```diff
@@ -10,6 +12,163 @@
+            try {
+                Thread.sleep(1000);
+            } catch (InterruptedException e) {
+                e.printStackTrace();
+            }
+            if (count == 0) {
+                //submitter.shutdown();
+                break;
+            }
+        }
+
+        System.out.print("Staring a new one submitter");
+
+        final DotSubmitter submitter2 =
+                dotConcurrentFactory.getSubmitter(submitterName);
+
+        System.out.println(submitter2);
+
+        assertTrue(submitter == submitter2);
+
+        IntStream.range(0, 20).forEach(
+                n -> {
+                    submitter2.execute(new PrintTask("Thread" + n));
+                }
+        );
+
+        //check active thread, if zero then shut down the thread pool
+        for (;;) {
+            int count = submitter2.getActiveCount();
+            System.out.println("Active Threads : " + count);
+            try {
+                Thread.sleep(1000);
+            } catch (InterruptedException e) {
+                e.printStackTrace();
+            }
+            if (count == 0) {
+                submitter2.shutdown();
+                break;
+            }
+        }
+
+    }
+
+    @Ignore
+    @Test
+    public void testDefaultOne_Submitter_Config_shutdown_keep_config() throws JSONException{
+
+        final String submitterName = "testsubmitter";
+        final DotConcurrentFactory dotConcurrentFactory =
+                DotConcurrentFactory.getInstance();
+
+        final DotSubmitter submitter =
+                dotConcurrentFactory.getSubmitter(submitterName,
+                        new SubmitterConfigBuilder().poolSize(2)
+                                .maxPoolSize(4).queueCapacity(500).build()
+                );
+
+        System.out.println(submitter);
+
+        IntStream.range(0, 40).forEach(
+                n -> {
+
+                    if (n % 10 == 0) {
+
+                        System.out.println(submitter);
+                    }
+                    submitter.execute(new PrintTask("Thread" + n));
+                }
+        );
+
+        assertEquals(2, submitter.getPoolSize());
+        assertEquals(4, submitter.getMaxPoolSize());
+
+        //check active thread, if zero then shut down the thread pool
+        for (;;) {
+            int count = submitter.getActiveCount();
+            System.out.println("Active Threads : " + count);
+            try {
+                Thread.sleep(1000);
+            } catch (InterruptedException e) {
+                e.printStackTrace();
+            }
+            if (count == 0) {
+                //submitter.shutdown();
+                break;
+            }
+        }
+
+        submitter.shutdown();
+
+        System.out.print("Staring a new one submitter");
+
+        final DotSubmitter submitter2 =
+                dotConcurrentFactory.getSubmitter(submitterName);
+
+        System.out.println(submitter2);
+
+        assertFalse(submitter == submitter2);
```

Comment:
Codacy found an issue: [Use assertSame(x, y) instead of assertTrue(x==y), or assertNotSame(x,y) vs assertFalse(x==y)](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6265363)